test/internet/test-dns-ipv6.js

```diff
@@ -40,6 +40,11 @@ function checkWrap(req) {
   assert.ok(typeof req === 'object');
 }
 
+if (!common.hasIPv6) {
```

Comment:
at least needs to be below these

https://github.com/john-yan/node/blob/master/test/internet/test-dns-ipv6.js#L10-L13